pywren_ibm_cloud/compute/backends/docker/docker.py

```diff
@@ -17,9 +18,24 @@
 from pywren_ibm_cloud.compute.utils import create_function_handler_zip
 
 logging.getLogger('urllib3.connectionpool').setLevel(logging.CRITICAL)
+logging.getLogger('paramiko.transport').setLevel(logging.CRITICAL)
 logger = logging.getLogger(__name__)
 
 
+class RemoteClientStartStopContextManager:
+    def __init__(self, remote_client=None):
+        self.remote_client = remote_client
+
+    def __enter__(self):
+        if self.remote_client:
+            self.remote_client.create_instance_action('start')
+            time.sleep(5)
```

Comment:
@omerb01 the last one...please make it confirable as well. We can't use 5 as static in the code